cores/esp8266/FSImpl.h

```diff
@@ -35,6 +35,7 @@ class FileImpl {
     virtual bool seek(uint32_t pos, SeekMode mode) = 0;
     virtual size_t position() const = 0;
     virtual size_t size() const = 0;
+    virtual int availableForWrite() { return 0; }
```

Comment:
There seem to be no afw in LittleFS.
If there is not, or if AFW is unsupported, what do you think of returning -1 or File::UNSUPPORTED or whatever < 0 ?

Reply:
The way I understand it, `availableForWrite()` means "how many bytes can be written w/o a potential blocking call."  And `0` is what the Arduino core is returning in the "I don't know case" (which is safe and agrees with the definition).

So, since this is for Arduino core compatibility, I'd say stick with 0.  If someone cares, they can dig into LittleFS's guts and figure out how to peek the temp sector usage and return it. :)

Reply:
Let's stick with Arduino API then. 0 = may block or (can't write <= unsupported)